### PR TITLE
Default versions now more reasonable

### PR DESCRIPTION
**Pull request by dekkerglen**

### CODE REVIEW

File: serverjs/cards.js [hunk 1]
```diff
@@ -115,6 +115,24 @@ function unloadCardDb() {
...
+  return notPromoOrDigitalCard(cardFromId(id));
+}
+
+function notPromoOrDigitalCard(card) {
+  return !card.promo && !card.digital && card.border_color != 'gold' && card.language == 'en' && card.tcgplayer_id;
+}
+
+function getMostReasonable(cardname) {
+  const cards = data.nameToId[cardname];
+  for (let i = 0; i < cards.length; i++) {
```
Comment (phulin) [inline serverjs/cards.js]:
```suggestion
  for (const card of cards) {
```

File: serverjs/cards.js [hunk 2]
```diff
@@ -115,6 +115,24 @@ function unloadCardDb() {
...
+    if (notPromoOrDigitalId(cards[i])) {
```
Comment (phulin) [inline serverjs/cards.js]:
```suggestion
    if (notPromoOrDigitalId(card)) {
```

File: serverjs/cards.js [hunk 3]
```diff
@@ -115,6 +115,24 @@ function unloadCardDb() {
...
+      return cardFromId(cards[i]);
```
Comment (phulin) [inline serverjs/cards.js]:
```suggestion
      return cardFromId(card);
```

File: serverjs/updatecards.js [hunk 4]
```diff
@@ -687,7 +689,15 @@ function convertCard(card, isExtra) {
   newcard.color_identity = newcard.color_identity.concat(card.color_identity);
   newcard.set = card.set;
   newcard.collector_number = card.collector_number;
-  newcard.promo = card.promo;
+  newcard.promo =
```
Comment (RKelln) [inline serverjs/updatecards.js]: I would put the promo detection in its own function to match the style of the `convertThing()` attributes and keep `convertCard()` as small as possible.

Comment (dekkerglen) [reply serverjs/updatecards.js]: I think I value not having these fields in the card object, to reduce the size of the actual object a lot more.

**Pull request merged**